lib/at_ex/gateway/sms.ex

```diff
@@ -61,4 +61,66 @@ defmodule AtEx.Gateway.Sms do
         {:error, message}
     end
   end
+
+  # Checkout token endpoints
+  @live_token_url "https://api.africastalking.com/checkout/token"
+  @sandbox_token_url "https://api.sandbox.africastalking.com/checkout/token"
+
+  # Using this system for delivery of which URL to use (sandbox or live) 
+  # determined by whether we ar in production or development or test
+  # Selection of the live URL can be forced by setting an environment
+  # variable FORCE_TOKEN_LIVE=YES
+  defp get_token_url do
```

Comment:
Could we have this used, on the other methods as well. I like your approach, of getting the URL are you open to if possible refactoring the other code to fetch the url using this method? instead of hardcoding as we had done?

Reply:
I'd love to.   Just assign me the issue.